.github/validation_samples/it_pileup/config.py

```diff
@@ -104,6 +104,43 @@
 hcal_veto = hcal.HcalVetoProcessor()
 hcal_veto.input_hit_pass_name = thisPassName
 
+# Load and configure  particle flow sequence. 
+# Here we use PF "tracking" and CLUE Ecal clustering 
+from LDMX.Recon import pfReco
+trackPF = pfReco.pfTrackProducer()
+trackPF.inputTrackCollName=trackPF.inputTrackCollName+overlayStr #"EcalScoringPlaneHitsOverlay" #                                                                                                                   
+trackPF.input_pass_name=inputPassName
+trackPF.doElectronTracking=True
+# reference info
+truthPF = pfReco.pfTruthProducer()
+    
+# CLUE     
+import LDMX.Ecal.ecalClusters as cl
+cluster = cl.EcalClusterProducer()
+cluster.seedThreshold = 350.                                                                                                                                                                             cluster.dc = 0.3
+cluster.nbrOfLayers = 1
+cluster.nbr_of_layers = cluster.nbrOfLayers #just here to highlight confusion
```

Comment:
We dont anymore have `nbrOfLayers` parameter
https://github.com/LDMX-Software/ldmx-sw/blob/trunk/Ecal/python/ecalClusters.py#L37
```suggestion
cluster.nbr_of_layers = 1
```

Reply:
great